dtfs-central-api/src/v1/routes/utilisation-reports-routes.js

```diff
@@ -355,8 +395,6 @@ utilisationReportsRouter
  *               $ref: '#/definitions/FeeRecordsToKeyResponseBody'
  *       400:
  *         description: Bad request
- *       404:
- *         description: Not Found
```

Comment:
Why has this been removed?

Reply:
Ah whoops I think I got a bit confused editing this file at some point, reverting